content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,194 @@
+
+    .. image:: plan_activities/schedule-list-activity.png
+       :align: center
+       :alt: List view of the CRM pipeline and the option to schedule an activity.
+
+   .. tab:: Activity view
+
+      Select a record on which to schedule an activity. Move across the row to find the desired
+      activity type, then click the :guilabel:`＋ (plus sign)`.
+
+    .. image:: plan_activities/schedule-activity-view.png
+       :align: center
+       :alt: Activity view of the CRM pipeline and the option to schedule an activity.
+
+View scheduled activities
+=========================
+
+To view scheduled activities, click the :guilabel:`🕘 (clock)` icon on the menu bar. Doing so opens
+the :guilabel:`Activities menu`. Activities are separated out by the application where they were
+created, and by deadline. Select an icon to see the activities for that specific application.
+
+.. image:: plan_activities/activities-menu.png
    :align: center
-   :height: 300
-   :alt: View of crm leads and the option to schedule an activity for Odoo Discuss
+   :alt: View of CRM leads page emphasizing the activities menu.
 
-Set your activity types
------------------------
+Configure activity types
+========================
 
-A number of activities types are available by default in Odoo (call, email, meeting, etc.). However,
-you can set new ones going to :menuselection:`Settings --> Activity types`.
+To configure the types of activities in the database, go to :menuselection:`Settings application -->
+Activities --> Activity Types`.
 
-.. image:: plan_activities/settings_activities_types.png
+.. image:: plan_activities/settings-activities-types.png
    :align: center
-   :height: 300
-   :alt: View of the settings page emphasizing the menu activity types for Odoo Discuss
+   :alt: View of the settings page emphasizing the menu activity types.
 
-.. tip::
-   If you need to create an activity type with an available calendar, make sure to create
-   it with an *Action to Perform* set as *Meeting*.
+To edit an existing activity type, select it from the list, then click :guilabel:`Edit`. To create a
+new activity type, click :guilabel:`Create`.
+
+Choose a :guilabel:`Name` for the new activity type.
 
-Recommend next activities
--------------------------
+.. image:: plan_activities/new-activity-type.png
+   :align: center
+   :alt: New activity type form.
+
+Activity settings
+-----------------
+
+Action
+~~~~~~
+
+*Actions* specify the intent of the activity. Some actions trigger specific behavior after an
+activity is scheduled.
+
+- If either **Phonecall** or **Meeting** are selected, users will have the option to open their
+  calendar to schedule a time for this activity.
+- If **Upload Document** is selected, a link to upload a document will be added directly to the
+  planned activity in the chatter.
+- If **Request Signature** is selected, a link will be added to the planned activity in the chatter
+  that will open a signature request pop-up.
```

Comment:
```suggestion
- If **Upload Document** is selected, a link to upload a document is added directly to the planned
  activity in the chatter.
- If either **Phonecall** or **Meeting** are selected, users have the option to open their calendar
  to schedule a time for this activity.
- If **Request Signature** is selected, a link is added to the planned activity in the chatter that
  opens a signature request pop-up window.
```
it might be best to explain the remaining options available on the Action drop-down menu, as well -- Reminder and Tax Report